Rakefile

```diff
@@ -9,12 +9,8 @@ namespace :bojangles do
     Bojangles.go!
   end
   task :daily do
```

Comment:
Apparently not. https://github.com/umts/bojangles/pull/35/commits/86cb0646dfd5a44d9583632b54d5385fd03d591c